msdk-featdet/msdk-featdet-ADAP-3D/src/main/java/io/github/msdk/featdet/ADAP3D/common/algorithms/SliceSparseMatrix.java

```diff
@@ -76,14 +80,14 @@
 	 *  mzValues is used to store all the mz values from raw file. 
 	 * </p>
 	 */
-	private final List<Integer> mzValues;
+	public final List<Integer> mzValues;
 	
 	/**
 	 * <p>
 	 * This is the data model for creating triplet representation of sparse matrix. 
```

Comment:
What is "triplet representation"? Remember that people who read your code need to understand it without knowing all the details of the algorithm beforehand.